controllers/tasksRequests.js

```diff
@@ -0,0 +1,60 @@
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const taskRequests = await taskRequestsModel.fetchTaskRequests();
+
+    if (taskRequests && taskRequests.length > 0) {
+      return res.status(200).json({
+        message: "Task requests returned successfully",
+        taskRequests,
+      });
+    }
+
+    return res.status(400).json({
+      message: "Unable to fetch task requests",
+    });
```

Comment:
should this be 400? if the taskRequest doesn't come for any other reasons? it iss not going to be only bad request I guess